models/models.go

```diff
@@ -63,6 +63,16 @@ type URLParams struct {
 	ResourceUUID uuid.UUID
 }
 
+// QueryParams for the /export/v1/exports endpoint
+type QueryParams struct {
+	Name        string    `json:"name"`
+	Created     time.Time `json:"created"`
+	Expires     time.Time `json:"expires"`
+	Status      string    `json:"status"`
+	Application string    `json:"application"`
+	Resource    string    `json:"resource"`
```

Comment:
Do we need to annotate these fields as json fields?  I don't think we do.  So it might be a bit more clear to their usage if we remove the json annotations.

Reply:
Makes sense, I made it a habit to put them, guess I forgot why. Thanks Derek!